final/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.8"
+services:
+  bitcoin-core:
+    build:
+      dockerfile: ./bitcoind/Dockerfile
+    image: bitcoind
+    container_name: bitcoind
+    command:
+      - --testnet
+      - -conf=/home/bitcoin/.bitcoin/bitcoin.conf
+      - -pid=/home/bitcoin/.bitcoin/testnet3/bitcoind-testnet.pid
+    volumes:
+      - data:/home/bitcoin/.bitcoin
+    expose:
+      - 8332
+      - 8333
+      - 18332
+      - 18333
+      - 18444
+    ports:
+      - 18332:18332
+      - 38332:38332
+      - 38333:38333
+    networks:
+      - LN_testing
+
+  core-lightning:
+    build:
+      dockerfile: ./lightningd/Dockerfile
+    image: lightningd
+    container_name: lightningd
+    command:
+      - --conf=/home/lightning/.lightning/testnet-config
+      - --bitcoin-rpcconnect=bitcoind
+      - --network=testnet
```

Comment:
can we move this argument to the `testnet-config` file?

Reply:
- `testnet` as we are specifying for `bitcoind` in `docker-compose` we should do that same here as well.
- `rpcconnect` depends on the network aliases present in `docker-compose`
- `conf` we can keep here or switch to `entrypoint`

Reply:
my comment was just about the `--network=testnet` argument.

we are already specifying a config file named "testnet-config` on line 33, so there's no reason to again specify testnet on line 35 - it's just duplication.  the `network=testnet` arg can go in the `testnet-config` file, since this file is specific for this network.

Reply:
I got the point of duplication so we have to eliminate it from one place. we can maybe replace it from `testnet-config` then so that the parameter passing for network can be done from `docker-compose` itself for both `bitcoind` and `core-lightnind`

Reply:
I'm not sure what you mean `testnet-config` doesn't currently have a network parameter (and I think it should).

I assume you plan to have multiple config files for CLN - one for each network, right?  then the network should be specified in each config file, so that `docker-compose.yml` can choose a network just by specifying the config file.

Reply:
Got it okay so i thought we would need to have a single config like bitcoind but yeah with multiple config files that solves it